packages/ui-toolkit/.storybook/main.js

```diff
@@ -5,5 +5,8 @@ module.exports = {
   "addons": [
     "@storybook/addon-links",
     "@storybook/addon-essentials"
-  ]
+  ],
```

Comment:
Why are we adding this ?

Reply:
This is needed for running the storybook at node v <16.20